src/error.rs

```diff
@@ -243,6 +256,20 @@ pub enum AxoassetError {
         /// The origin path of the asset, used as an identifier
         origin_path: String,
     },
+
+    /// This error indicates we ran into an issue when creating an archive.
+    #[error("failed to create archive: {reason}")]
+    #[diagnostic(help(
```

Comment:
same, not sure this is the right diagnostic (can a user take action to fix this?)